scripts/manual_operations/manual_remove.py

```diff
@@ -10,37 +10,31 @@
 from __future__ import print_function
 import argparse
 
-from utils.clients.database_client import DatabaseClient
+from utils.clients.django_database_client import DatabaseClient
 
 
 class ManualRemove:
     """
     Handles removing a run from the database
     """
 
-    def __init__(self, instrument, credentials=None):
-        if credentials:
-            self.database = DatabaseClient(credentials)
-        else:
-            self.database = DatabaseClient()
+    def __init__(self, instrument):
+        self.database = DatabaseClient()
         self.database.connect()
         self.to_delete = {}
         self.instrument = instrument
 
     def find_runs_in_database(self, run_number):
         """
-        Find all runs in the database that relate to a given instrument and run number
+        Find all run versions in the database that relate to a given instrument and run number
         :param run_number: the run to search for in the database
```

Comment:
Would it be worth adding the datatype to the parameters and return values in docstrings to be more explicit with method usage? 
`:param run_number: (int) the run to search for in the database`